### ISSUE TITLE
Reduce the intensity of the limiter

### ISSUE THREAD

**Issue by jasonbrianmerrill** [sergree/matchering#65]
Hello,  this device is wonderful, and I am sorry if this isn't really an "issue" but I find that the limiting can sometimes be too strong on some results - maybe I can reduce the volume of the reference file? Or the target file? 

I know it might not work but it would be amazing to have a mix slider or intensity (of the limiter) slider or parameter change.

I know this is probably outside of the scope of this project or maybe just not the intended workflow - but i think it would be amazingly usable for me for even a final result. Perhaps some pre-manipulation would work.

**Comment by Analog4Lyfe** [comment]
Yea I notice it with bass. It’s very heavy on pulling the bass.

**Comment by jasonbrianmerrill** [comment]
why would trying to hack me solve it? lol

**Comment by deton24** [comment]
You can do it in e.g. [ComfyUI-Matchering](https://github.com/MuziekMagie/ComfyUI-Matchering).

**Comment by jasonbrianmerrill** [comment]
> You can do it in e.g. [ComfyUI-Matchering](https://github.com/MuziekMagie/ComfyUI-Matchering). Standalone/portable ComfyUI package for CPU or Nvidia: [new_ComfyUI_windows_portable_nvidia_cu124_or_cpu.7z](https://github.com/comfyanonymous/ComfyUI/releases/download/latest/new_ComfyUI_windows_portable_nvidia_cu124_or_cpu.7z)

Wow, very cool. Thank you.

Now, all Im looking for is some kind of "mix" knob - to mix the eq spectra of both of these. Basically a way to adjust the strength of certain matchering processes. Maybe is some functionality I'm not seeing.

This is already a GREAT step. Thanks again.

**Comment by deton24** [comment]
Maybe import both tracks into DAW, align and adjust their volume levels to
your liking. If phase is not changed in the output file so waveform looks
similar or the same, it will work.

śr., 16 kwi 2025, 19:24 użytkownik jasonbrianmerrill <
***@***.***> napisał:

**Comment by jasonbrianmerrill** [comment]
yes! Very true. Do you think there would be any phase relationships changed? There are free tools to check the phase, but just wondering :)

**Comment by jasonbrianmerrill** [comment]
I also wonder if instead of a straight up mix knob, it could be the model "mixing" the "qualities" of each file, say, creating something in the middle. But, as it is (especially with some advanced settings revealed in ComfyUI's version) it's fantastic.

**Comment by deton24** [comment]
If you decrease the volume of both tracks by 3dB, you'll have the equal
amount of every track in the output.
If there are any phase issues preventing you from such a combination,
you'll immediately know that by some audible distortion, sounding similarly
like you weren't able to align the files, so various frequencies are
cancelling and overlapping each other.

czw., 17 kwi 2025 o 06:18 jasonbrianmerrill ***@***.***>
napisał(a):
